src/Symfony/Component/DependencyInjection/Tests/Fixtures/includes/autowiring_classes.php

```diff
@@ -252,7 +254,8 @@ public function setFoo(Foo $foo)
         // should be called
     }
 
-    /** @inheritdoc*/ // <- brackets are missing on purpose
+    /** {@inheritdoc}*/ // <- brackets are missing on purpose
```

Comment:
Sure about that? 🤔

Reply:
Blindly applying fabbot's patches ☺️. I revert...